### DISCUSSION TITLE
Unable to use Alexa on GTS3

### DISCUSSION THREAD

**Discussion by vlaraort** [zepp-health/zeppos-forum#8] [User Feedback]
I am having problems with Alexa in GTS3.
When I open the Zepp app and the watch gets synchronized, it works fine, but after some minutes, if I try to use it, I just get a error message in the watch "Synchronize the watch with the application" and I cannot use it anymore until I reopen Zepp and let it get sync, then it works until next time.
My zepp app has all permissions granted, open, battery optimization disabled... 
Any ideas if it can be fixed? It have no sense to have to open zepp and wait until the watch get sync before using it.
Thanks!

**Comment by MyoungXUE** [comment]
Hello, thanks for your support of the forum!

This problem is a usage problem, and I have classified it under the User Feedback category.

I'm sorry that our product is causing you problems, I will send this question to the relevant colleagues.

**Reply by vlaraort** [reply]
Okay, thank you so much for the quick response! One of the reasons that I had to update from GTS1 to GTS3 was the alexa support, and if I have to open the phone + zepp app + wait synchronization, every time that I want to use it, the feature loses all it's sense :)
